.kiro/specs/smile-zemi-crawler-line-notify/design.md

```
@@ -765,6 +765,218 @@ interface ArtifactData {
+
+#### 3. セレクタの段階的特定
+
+各ステップでDOM構造を調査し、適切なセレクタ（CSS Selector、XPath、テキストコンテンツ等）を特定して実装に反映します。
+
+**Selector Strategy**:
+- **Primary Selector**: 最も安定性が高いセレクタ（ID、data属性）
+- **Fallback Selectors**: 代替セレクタ（class、要素タイプ、テキストコンテンツ）
+- **Dynamic Selectors**: 動的生成要素用の待機ロジック
+
+**Selector Definition** (`src/config/selectors.js`):
+```javascript
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)